libs/wire-api/src/Wire/API/User.hs

```diff
@@ -802,6 +835,24 @@ data NewTeamUser
   deriving stock (Eq, Show, Generic)
   deriving (Arbitrary) via (GenericUniform NewTeamUser)
 
+newTeamUserCode :: NewTeamUser -> Maybe InvitationCode
```

Comment:
These can be autogenerated by `makePrisms`. For example, instead of `newTeamUserCode` you can use `preview _NewTeamMember`.